man/homectl.xml

```diff
@@ -963,6 +963,28 @@
 
         <xi:include href="version-info.xml" xpointer="v245"/></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><option>--session=</option><replaceable>SESSION</replaceable></term>
```

Comment:
Shouldn't this be --session-file to match the json field? Otherwise it might get ambiguous if other session thingys are added in the future

Reply:
yeah, please call this `--session-file=`